src/audio/kpb.c

```diff
@@ -62,7 +62,7 @@ struct comp_data {
 /*! KPB private functions */
 static void kpb_event_handler(int message, void *cb_data, void *event_data);
 static int kpb_register_client(struct comp_data *kpb, struct kpb_client *cli);
-static void kpb_init_draining(struct comp_data *kpb, struct kpb_client *cli);
```

Comment:
Commit message needs to say why you are doing this.

Reply:
Updated.